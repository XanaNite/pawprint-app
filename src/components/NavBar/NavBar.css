.hamburger-menu{
    font-size: 24px;
    float: left;
    margin-right: 0;
    margin-left: 15px;
}

nav {
    display: inline;
}

.navigation {
    float: left;
    position: fixed;
    background: white;
    width: 100%;
    z-index: 1;
    padding: 20px 0px;
    margin-bottom: 100px;

}