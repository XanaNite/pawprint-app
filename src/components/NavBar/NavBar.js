import React from 'react'
import './NavBar.css'
import Menu from '../Menu/Menu'
import Logo from '../Logo/Logo'

export default function NavBar(){
    return (
        <nav role='navigation'>
            <Menu />
            <Logo />
        </nav>
    )
}

