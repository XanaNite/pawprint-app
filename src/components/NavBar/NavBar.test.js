import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import NavBar from './NavBar';

describe(`NavBar component`, () => {
  const props = {
    className: 'test-class-name',
    children: <p>test children</p>,
    'data-other': 'test-other-prop'
  };

  it('renders NavBar by default', () => {
    const wrapper = shallow(<NavBar />)
    expect(toJson(wrapper)).toMatchSnapshot()
  });

  it('renders the NavBar given props', () => {
    const wrapper = shallow(<NavBar {...props} />)
    expect(toJson(wrapper)).toMatchSnapshot()
  });
})
