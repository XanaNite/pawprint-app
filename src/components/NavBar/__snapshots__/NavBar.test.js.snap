// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavBar component renders NavBar by default 1`] = `
<nav
  role="navigate"
>
  <div
    className="navigation"
  >
    <div
      className="menuNav"
    >
      <Menu />
    </div>
    <div
      className="logoNav"
    >
      <Logo />
    </div>
  </div>
</nav>
`;

exports[`NavBar component renders the NavBar given props 1`] = `
<nav
  role="navigate"
>
  <div
    className="navigation"
  >
    <div
      className="menuNav"
    >
      <Menu />
    </div>
    <div
      className="logoNav"
    >
      <Logo />
    </div>
  </div>
</nav>
`;
